src/System.Management.Automation/FormatAndOutput/common/PSStyle.cs

```diff
@@ -23,11 +23,23 @@ public enum OutputRendering
     }
     #endregion OutputRendering
 
+    /// <summary>
+    /// Defines the options for views of progress rendering.
+    /// </summary>
+    public enum ProgressView
+    {
+        /// <summary>Render progress using minimal space.</summary>
+        Minimal = 0,
+
+        /// <summary>Class rendering of progress.</summary>
+        Full = 1,
+    }
```

Comment:
Since it is a public can we have more informative names for the enum and members?
Really the enum defines a _style_. I propose `ProgressViewStyle` name. 
For members:
Minimal -> OneLine
Full -> Classic (?)

The descriptions could be updated too.

Reply:
It's one line per progress instance, but can be multiple lines with child and sibling progress reports.  Full changing to Classic would be fine (that's a typo I just noticed in the description which was supposed to say `Classic`.  Will update.